parsl/monitoring/web_app/index.py

```diff
@@ -0,0 +1,62 @@
+import dash_core_components as dcc
+import dash_html_components as html
+from dash.dependencies import Input, Output
+from parsl.monitoring.web_app.app import app, init_db
+import argparse
+
+
+def web_app(db, port):
+    if not init_db(db):
+        return
+
+    print(' * Visualizing ' + db.split('/').pop())
+    print(' * Running on http://localhost:' + str(port) + '/workflows')
+
+    from parsl.monitoring.web_app.apps import sql, workflows, tabs
+
+    app.layout = html.Div([
+        dcc.Location(id='url', refresh=False),
+        # html.Nav(children=[
+        #     dcc.Link('Workflows', href='/workflows'),
+        #     dcc.Link('SQL', href='/sql')]),
+        html.Div(id='page-content')
+    ])
+
+
+    @app.callback(Output('page-content', 'children'),
+                  [Input('url', 'pathname')])
+    def display_page(pathname):
+        if pathname == '/': # TODO Redirect to workflows or show special page
+            pass
+        elif pathname == '/workflows':
+            return workflows.layout
+        elif '/workflows' in str(pathname):
+            return tabs.display_workflow(workflow_name=pathname.split('/').pop())
```

Comment:
is this going to behave weirdly if workflow_name has a / in it? can it have a / in it?